test/micro-foo.test.ts

```diff
@@ -69,23 +70,25 @@ ava('Basic usage, create http server', async (t: Assertions) => {
 	const output = await getLogsFromFile(file.path);
 
 	// Logs on stdout
-	t.true(output[0].includes('Init module bar'), 'Init module bar');
-	t.true(output[1].includes('Init module foo'), 'Init module foo');
-	t.true(output[2].includes('Hello foo.init'), 'Hello foo.init');
-	t.true(output[3].includes('Listening on port 5000'), 'Listening on port 5000');
-	t.true(output[4].includes('GET /foo'), 'GET /foo');
-	t.true(output[5].includes('GET /'), 'GET /');
-	t.true(output[6].includes('GET /ping'), 'GET /ping');
-	t.true(output[7].includes('GET /version'), 'GET /version');
-	t.true(output[8].includes('not-found'), `not-found ${output[8]}`);
-	t.true(output[9].includes('  err: {'), `Error details on multi-lines ${output[9]}`);
-	t.true(output[10].includes('"type": "N9Error",'), `Error details on multi-lines ${output[10]}`);
+	t.true(output[0].includes('Conf loaded: development'));
+	t.true(output[1].includes('Configuration validation is disabled'));
```

Comment:
Here the message related to the assertion is missing. This is helpfull to debug tests errors.